tasks/discover-pdrs/index.js

```diff
@@ -10,18 +10,33 @@ const { runCumulusTask } = require('@cumulus/cumulus-message-adapter-js');
  * Fetch a list of files from the provider
  *
  * @param {Object} params
- * @param {Object} params.providerConfig     - the connection config for the provider
- * @param {bool}   params.useList            - flag to tell ftp server to use 'LIST'
- *                                             instead of 'STAT'
- * @param {number} params.httpRequestTimeout - seconds for http provider to wait
- *                                             before timing out
- * @param {*}      path                      - the provider path to search
- * @returns {Array<Object>}                  - a list of discovered file objects
+ * @param {Object} params.providerConfig - the connection config for the
+ *   provider
+ * @param {bool} params.useList - flag to tell ftp server to use 'LIST'
+ *   instead of 'STAT'
+ * @param {number} [params.httpRequestTimeout=300] - seconds for http provider
+ * to wait before timing out
+ * @param {*} params.path - the provider path to search
```

Comment:
We don't know the type of `params.path`?

Reply:
0cfb14418